ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -196,7 +196,19 @@
     set_fact:
       generate_config: "{{ use_custom_config and (not master_config_stat.stat.exists or not console_ssl_stat.stat.exists) }}"
 
-  - name: Create command line for oc cluster up execution
+  - name: Create command line for oc cluster up execution for older oc clients, up to 3.9
+    set_fact:
+      oc_cluster_up_cmd: >-
+        {{ oc_cmd }} cluster up
+        --routing-suffix={{ openshift_routing_suffix }}
+        --public-hostname={{ hostname }}
+        --host-pv-dir={{ persistent_vol_mount_point }}
```

Comment:
Based on testing on Mac I needed to add this line back in.
I want to test this more to determine why...I forgot the original reason of breaking this out.  
Felt it might be something with how Mac runs, oc is running on a docker VM, think with this we wanted to force the PVs to exist on the Mac host and not docker VM...but memory is a little fuzzy.

Reply:
Do we need to add a `when` statement for versions below 3.10? @jmontleon made comments about this not working in 3.10.

Reply:
The command gets overwritten, so technically we are building up a broken command and then overwriting it in next step for 3.10 and newer.

Longer term, we should come back and refactor as you said to clean things up with the different versions.

For now I was trying to get functionality restored making minimal changes.